README.md

```diff
@@ -111,13 +111,16 @@ Tested with the following software:
 
 Install dependencies (`sudo apt install libopencv-dev build-essential v4l2loopback-dkms curl`).
 
+Clone this repository with `git clone --recursive --depth=1 https://github.com/floe/deepbacksub.git`.
```

Comment:
Using shallow clones comes with its quite own downsides (even though most of them have been addressed in recent `git` releases). Please drop the `--depth=1` in the example snippet. It can be mentioned in a second sentence, but shouldn't be the default that we document.

Reply:
Okay, I've adapted README accordingly.